.github/workflows/manual-deployment.yml

```diff
@@ -0,0 +1,106 @@
+name: Manual Deploy to Maven Central
+# Trigger this manual deployment in scenarios where:
+# - A git tag exists, but the initial deployment failed.
+# - Fixes have been pushed to the main branch.
```

Comment:
I think this is already handled? Fixes to `main` branch already trigger a release with new tag? 🤔